spec/factories/category.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :category do
+    sequence(:key) { |n| "category_#{n}" }
+  end
+
+  factory :category_translation do
+    sequence(:name) {|n| "Category translation #{n}"}
+    sequence(:key) {|n| "category_translation_#{n}"}
```

Comment:
I believe the `key` here is duplicated from `category`, so your factory should mimic this behavior.